bin/printAlertStream.py

```diff
@@ -9,9 +9,34 @@
 from __future__ import print_function
 import argparse
 import sys
+import os
 from lsst.alert.stream import alertConsumer
 
 
+def msg_text(message):
+    """Remove postage stamp cutouts from an alert message.
+    """
+    message_text = {k: message[k] for k in message if k not in ['cutoutDifference', 'cutoutTemplate']}
```

Comment:
I will have to think about this. (Also, the schema files here are only temporarily hardcoded in the command line scripts- later we can use a schema registry to distribute schemas.) If using Avro and the writer schema needs changing, we can update it without breaking the reader, even if the reader does not have the updated schema. The readers can have their own schemas that can be significantly different from the writers (different key names, etc.) as long as the data fits (which will be a nice feature for filtering down the line) so I don't think we want to go down the line of enforcing too much learning from the original schema.